Libraries/LibHTML/DOM/Element.cpp

```diff
@@ -160,3 +162,26 @@ void Element::recompute_style()
         layout_node()->set_needs_display();
     }
 }
+
+RefPtr<StyleProperties> Element::computed_style()
```

Comment:
If this function can never return null, it should return a `NonnullRefPtr<StyleProperties>`.